pending/dancer2-plugin-paginator.pod

```diff
@@ -0,0 +1,125 @@
+=head1 HISTORY
```

Comment:
OTTOMH, the `=head1` title should be the one displayed when listing all articles for a year, and `=head2` and below used for sub-sections of the article - so it would be better to start with e.g. `=head1 Dancer2::Plugin::Paginator reborn` or something, and move the `=head1`'s to `=head2`'s.